### DISCUSSION TITLE
How to permanently enable this bar (what is it called?)

### DISCUSSION THREAD

**Discussion by sirlaughalot90** [chrisgrieser/shimmering-focus#182] [Q&A] [answered]
Is there a way to always show this bar (On both sides, meaning the bar with File Explorer, Search, Favorites and on the right)?

<img width="365" alt="CleanShot 2022-07-07 at 13 15 34@2x" src="https://user-images.githubusercontent.com/20352339/177761098-be5f6e42-d93d-4e9b-9857-b32de8197802.png">

Thank you. 😊

**Comment by chrisgrieser** [comment, marked as answer]
they are called sidebar tabs, and it's this setting here :)

![image](https://user-images.githubusercontent.com/73286100/177777368-3c30f635-bbc7-4b03-8fe2-e933a5e1897c.png)

**Reply by sirlaughalot90** [reply]
Now I feel stupid. 😅 I was 100% sure that I tried every single switch.. oh well. Thank you! 😄
